.eslintrc.js

```diff
@@ -43,15 +45,6 @@ module.exports = {
       rules: Object.assign({}, require('eslint-plugin-node').configs.recommended.rules, {
         // add your custom rules and overrides for node files here
       })
-    },
-
-    // test files
-    {
-      files: ['tests/**/*.js'],
-      excludedFiles: ['tests/dummy/**/*.js'],
-      env: {
-        embertest: true
-      }
```

Comment:
This doesn't mean that linting's been removed from our test files does it? 😬

Reply:
no no

Reply:
yeah, this is just from the ember update, not our changes

Reply:
OK cool, thanks!